notification-manager/src/main/java/com/qpay/notificationmanager/service/impl/EmailNotificationService.java

```diff
@@ -3,14 +3,25 @@
 import com.qpay.libs.models.PaymentNotification;
 import com.qpay.notificationmanager.service.NotificationService;
 import lombok.RequiredArgsConstructor;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
 import org.springframework.stereotype.Service;
 
 @Service
 @RequiredArgsConstructor
 public class EmailNotificationService implements NotificationService {
 
-    public PaymentNotification sendMessage(final PaymentNotification paymentNotification) {
-        return paymentNotification;
+    private final JavaMailSender javaMailSender;
```

Comment:
```suggestion
    private final JavaMailSender mailSender;
```

What you want is to send messages. You don't need to show what implementation exactly you use javaMailSender or netflix or anything else.